src/test/java/com/spatial4j/core/io/GeneralWktTest.java

```diff
@@ -58,6 +58,11 @@ protected ShapeWriter getShapeWriterForTests() {
     return writerForTests;
   }
 
+  @Override
+  public void testWriteThenReadBufferedLine() throws Exception {
```

Comment:
This is sad; please see https://github.com/locationtech/spatial4j/pull/124  in which I was able to remove this override.  Perhaps you can help finish that; I have no time left today.